backend/api/serializers.py

```diff
@@ -282,19 +282,19 @@ class AccessListSerializer(serializers.ModelSerializer):
     submission_status = serializers.SerializerMethodField()
 
     def get_auditee_uei(self, access):
-        return access.sac.auditee_uei
+        return access.audit.auditee_uei
```

Comment:
I think we're going to want to also check the sac as well. Since a lot of the audits will be null on first push. 

Note: 
(i) We'll want to make the same change throughout this file
 (ii) We'll want to update tests to validate this as well.
 

```suggestion
        return access.audit.auditee_uei if access.audit else access.sac.auditee_uei
```